sammba/registration/struct.py

```diff
@@ -596,7 +611,7 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
                     weight=nonlinear_weight_file,
                     iniwarp=[warp_file],
                     inilev=nonlinear_levels[-1] + 1, # not ideal
-                    minpatch=level_or_minpatch,
+                    minpatch=nonlinear_minimal_patches,
```

Comment:
nonlinear_minimal_patches is a list. Here you need to put `minpatch=minpatch`

Reply:
There is an indentation problem [here](https://github.com/sammba-mri/sammba-mri/blob/a501437a0df321404f8c3f1d592305ebdfd60102/sammba/registration/struct.py#L606)